### PR TITLE
Problems solved on 9/25

### PR DESCRIPTION
**Pull request by HidemichiShimura**
These are problems on LeetCode which were solved on 9/25.

### CODE REVIEW

File: 1464.maximum-product-of-two-elements-in-an-array.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+var maxProduct = function(nums) {
+    let maxValue = 0;
+    
+    for (let i = 0; i < nums.length; i++) {
+        for (let j = i+1; j < nums.length; j++) {
+            const currValue = (nums[i]-1) * (nums[j]-1);
+            if (maxValue < currValue) {
+                maxValue = currValue;
+            }
```
Comment (ryonryon) [inline 1464.maximum-product-of-two-elements-in-an-array.js:18]:
```suggestion
            maxValue = Math.max(maxValue, currValue);
```

Do you think it is making easy to read?

File: 1725.number-of-rectangles-that-can-form-the-largest-square.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            currLen = rectangles[i][0];
+        } else {
+            currLen = rectangles[i][1];
+        }
+
+        squares.push(currLen);
+        
+        if (maxLen < currLen) {
+            maxLen = currLen;
+        }
```
Comment (ryonryon) [inline 1725.number-of-rectangles-that-can-form-the-largest-square.js:26]: same here. you can use Math.max()

File: 760.find-anagram-mappings.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @param {number[]} nums1
+ * @param {number[]} nums2
+ * @return {number[]}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/find-anagram-mappings/
+*/
+
+var anagramMappings = function(nums1, nums2) {
```
Comment (ryonryon) [inline 760.find-anagram-mappings.js:11]: Can you use Array.prototype.map() to improve this?

**Review by ryonryon — APPROVED** [review]

File: 1464.maximum-product-of-two-elements-in-an-array.js (hunk 1 above)
Comment (HidemichiShimura) [reply 1464.maximum-product-of-two-elements-in-an-array.js:18]: I modified the code as the suggestion above.

File: 1725.number-of-rectangles-that-can-form-the-largest-square.js (hunk 2 above)
Comment (HidemichiShimura) [reply 1725.number-of-rectangles-that-can-form-the-largest-square.js:26]: I modified the code with Math.max().

File: 760.find-anagram-mappings.js (hunk 3 above)
Comment (HidemichiShimura) [reply 760.find-anagram-mappings.js:11]: I modified the code with array.map().

**Comment by HidemichiShimura** [comment]
Thank you for your review.

**Pull request merged**